streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/ThreadMetricsTest.java

```diff
@@ -298,41 +297,4 @@ public void shouldGetSkipRecordSensor() {
         assertThat(sensor, is(expectedSensor));
     }
 
-    @Test
-    public void shouldGetCommitOverTasksSensor() {
-        final String operation = "commit";
-        final String operationLatency = operation + StreamsMetricsImpl.LATENCY_SUFFIX;
-        final String totalDescription =
-            "The total number of calls to commit over all tasks assigned to one stream thread";
-        final String rateDescription =
-            "The average per-second number of calls to commit over all tasks assigned to one stream thread";
-        final String avgLatencyDescription =
-            "The average commit latency over all tasks assigned to one stream thread";
-        final String maxLatencyDescription =
-            "The maximum commit latency over all tasks assigned to one stream thread";
-        expect(streamsMetrics.threadLevelSensor(THREAD_ID, operation, RecordingLevel.DEBUG)).andReturn(expectedSensor);
-        expect(streamsMetrics.taskLevelTagMap(THREAD_ID, ROLLUP_VALUE)).andReturn(tagMap);
-        StreamsMetricsImpl.addInvocationRateAndCountToSensor(
-            expectedSensor,
-            TASK_LEVEL_GROUP,
-            tagMap,
-            operation,
-            rateDescription,
-            totalDescription
-        );
-        StreamsMetricsImpl.addAvgAndMaxToSensor(
-            expectedSensor,
-            TASK_LEVEL_GROUP,
-            tagMap,
-            operationLatency,
-            avgLatencyDescription,
-            maxLatencyDescription
-        );
-        replay(StreamsMetricsImpl.class, streamsMetrics);
-
-        final Sensor sensor = ThreadMetrics.commitOverTasksSensor(THREAD_ID, streamsMetrics);
-
-        verify(StreamsMetricsImpl.class, streamsMetrics);
-        assertThat(sensor, is(expectedSensor));
-    }
```

Comment:
req: Please keep this test an do only remove the following part (and of course the parts that become obsolete):
```
StreamsMetricsImpl.addAvgAndMaxToSensor(
            expectedSensor,
            TASK_LEVEL_GROUP,
            tagMap,
            operationLatency,
            avgLatencyDescription,
            maxLatencyDescription
        );
```